hrjobcontract/CRM/Hrjobcontract/BAO/HRJobContractRevision.php

```diff
@@ -113,7 +114,7 @@ public static function updateEffectiveEndDates($jobcontractId) {
    * @param int $contactID Contact ID to validate against
    * @param string $effectiveDate Date in Y-m-d format
    *
-   * @return array Array in the following format :
+   * @return array array in the following format :
```

Comment:
It's not duplicated. The first array is the type of the param and the second one is part of the description: "Array in the following format"